iac/compositions/cat-full/service_cat_api.tf

```diff
@@ -198,3 +198,32 @@ resource "aws_security_group_rule" "cat_api_lb__public_https_in" {
   type              = "ingress"
 }
 
+resource "aws_security_group" "cat_api_clients" {
```

Comment:
I've already added this and the LB rules in my open PR (https://github.com/Crown-Commercial-Service/ccs-scale-cat-paas-infra/pull/48)

Reply:
removing